plugins/smart_proxy_dynflow_core/rules

```diff
@@ -11,5 +11,9 @@ export DH_RUBY_IGNORE_TESTS=require-rubygems
 # If you need to specify the .gemspec (eg there is more than one)
 #export DH_RUBY_GEMSPEC=gem.gemspec
 
+build:
+	/bin/sed -ri '/^ExecStart/ s#/usr/bin/smart_proxy_dynflow_core#/usr/bin/bundle exec smart_proxy_dynflow_core#' deploy/smart_proxy_dynflow_core.service
+	/bin/sed -ri '/^EnvironmentFile/ s#sysconfig#default#'
```

Comment:
```
12:25:47 /bin/sed -ri '/^EnvironmentFile/ s#sysconfig#default#'
12:25:47 /bin/sed: no input files
```

```suggestion
	/bin/sed -ri '/^EnvironmentFile/ s#sysconfig#default#' deploy/smart_proxy_dynflow_core.service
```

Reply:
Yeah, sometimes I should stop and read the stuff I wrote...